tests/Unit/Instrumentation/AwsSdk/AwsSdkInstrumentationTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Test\Unit\Instrumentation\AwsSdk;
+
+use OpenTelemetry\Instrumentation\AwsSdk\AwsSdkInstrumentation;
+use PHPUnit\Framework\TestCase;
+
+class AwsSdkInstrumentationTest extends TestCase
+{
+    public function testInstrumentationClassName()
+    {
+        $awsSdkInstrumentation = new AwsSdkInstrumentation();
+
+        $this->assertEquals(
+            'AWS SDK Instrumentation',
+            $awsSdkInstrumentation->getName()
+        );
```

Comment:
This could be one call:
```php
        $this->assertEquals(
            'AWS SDK Instrumentation',
            (new AwsSdkInstrumentation())->getName()
        );

```